Rakefile

```diff
@@ -10,6 +14,12 @@ task :spec do
   Rake::Task['test:spec'].invoke
 end
 
+task :acceptance do
+  Rake::Task['test:base'].invoke
```

Comment:
Wondering if this could just be done by listing these as dependencies on the task, rather than explicit `invoke` calls?